build/stf-run-ci/tasks/create_builds.yml

```diff
@@ -2,10 +2,13 @@
 - name: Create BuildConfig and ImageStream
   shell: oc new-build --name {{ artifact.name }} --dockerfile - < {{ artifact.working_build_dir }}/{{ artifact.dockerfile_path }}
 
+- name: Kill first build since it will always fail (triggered on BuildConfig creation)
```

Comment:
Why does it always fail?

Reply:
Because when you do a BuildConfig it creates a Build automatically (and you can't squelch that unfortunately). It seems to fail on importing the contents of the repo as a binary and fails the Build. I could never figure out why it failed like that, but it seems like I wasn't the only one to have that issue. It has something to do with how the binary/contents of the source are copied in during `BuildConfig` creation.

Doing any subsequent builds with `oc start-build` seems to result in successful builds though.